src/main/kotlin/org/rust/ide/refactoring/introduceUtils.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.refactoring
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiNamedElement
+import com.intellij.refactoring.introduce.inplace.InplaceVariableIntroducer
+
+class RustInPlaceVariableIntroducer(
```

Comment:
Let's use `Rs` prefix for such names to be consistent with all other code
And maybe it's worth to rename the file to `RsInPlaceVariableIntroducer`? It contains only this class